pybamm/__init__.py

```diff
@@ -223,7 +223,7 @@ def version(formatted=False):
 #
 # other
 #
-from .quick_plot import QuickPlot, dynamic_plot, ax_min, ax_max
+from .quick_plot import QuickPlot, plot, plot2D, dynamic_plot, ax_min, ax_max
```

Comment:
might be worth making a new plotting folder with a few different functions in different files?

Reply:
agreed